suite-native/module-accounts-import/src/accountsImportThunks.ts

```diff
@@ -36,13 +32,8 @@ const getAccountTypeFromDescriptor = (
 export const importAccountThunk = createThunk(
     `${actionPrefix}/importAccountThunk`,
     ({ accountInfo, accountLabel, coin }: ImportAssetThunkPayload, { dispatch, getState }) => {
-        const device = selectDeviceById(getState(), HIDDEN_DEVICE_ID);
         const deviceState = HIDDEN_DEVICE_STATE;
 
-        if (!device) {
```

Comment:
Q: Wouldn't be "safer" to keep check here? I am not sure just asking for friend

Reply:
Hmmm 🤔 The device gets created with app initialization so we are certain the device will be there. And we don't actually need the device here, we just need to create the account with `deviceState`. That, as far as I understand is a unique hash of devices so when we have this, even if we created that device later when we need it, it would map onto the device when selecting it's accounts. If that makes sense? 

Basically here we don't need the device, but we're sure it's already created.